pkg/R/RDD.R

```diff
@@ -1374,6 +1374,42 @@ setMethod("top",
             takeOrderedElem(rdd, num, FALSE)
           })
 
+#' Pipes elements to a forked externel process.
+#'
+#' The same as `pipe()' in Spark.
+#'
+#' @param rdd the RDD whose elements are piped to the forked externel process.
+#' @param command the command to fork an externel process.
+#' @param ... other arguments passed to pipeRDD.
+#' @param env character vector of name=value strings to set environment variables.
+#' @return a new RDD created by piping all elements to a forked externel process.
+#' @rdname pipeRDD
+#' @export
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' rdd <- parallelize(sc, 1:10)
+#' collect(pipeRDD(rdd, "cat")
+#' Output: c("1", "2", ..., "10")
+#'}
+setGeneric("pipeRDD", function(rdd, command, ...) { standardGeneric("pipeRDD") })
+
+#' @rdname pipeRDD
+#' @aliases pipeRDD,RDD,character-method
+setMethod("pipeRDD",
+          signature(rdd = "RDD", command = "character"),
+          function(rdd, command, env = character()) {
+            func <- function(part) {
+              trim.trailing.func <- function(x) {
+                sub("\\s+$", "", as.character(x))
```

Comment:
Recommend to use toString() instead of as.character(). I think this is more representational and consistent with pyspark.
For example, if an RDD element x is list(1,2,3),
> x<-list(1,2,3)
> as.character(x)
[1] "1" "2" "3"
> toString(x)
[1] "1, 2, 3"

In python,
In [1]: x=[1,2,3]

In [2]: str(x)
Out[2]: '[1, 2, 3]'